### PR TITLE
add tests for API and database

### PR DESCRIPTION
**Pull request by DVA09**

### CODE REVIEW

File: Database/test/specs/database.test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  before(function () {
+    client.connect();
+    client.query(
+      "INSERT INTO pets (id, name) VALUES (1, 'Bentley'), (2, 'Mari'), (3, 'Tank'), (4, 'Marley'), (5, 'Milo'), (6, 'Gunner'), (7, 'Louie'), (8, 'Thor'), (9, 'Cooper'), (10, 'Leo')"
+    );
+  });
+
+  after(function () {
+    client.query("TRUNCATE TABLE pets");
+  });
```
Comment (Alex19777) [inline Database/test/specs/database.test.js]: Try to create separate class for this methods and name it hooks

File: Database/test/pageobjects/login.page.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const { $ } = require('@wdio/globals')
+const Page = require('./page');
+
+/**
+ * sub page containing specific selectors and methods for a specific page
+ */
+class LoginPage extends Page {
```
Comment (Alex19777) [inline Database/test/pageobjects/login.page.js]: Why do you have UI tests in DataBase folder?

File: Database/test/specs/database.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+const axios = require("axios");
+const { Client } = require("pg");
+const pet = require("../utils/pet.json");
+const client = new Client({
```
Comment (Alex19777) [inline Database/test/specs/database.test.js]: Move DB client to separate class

File: Database/test/specs/database.test.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    );
+  });
+
+  after(function () {
+    client.query("TRUNCATE TABLE pets");
+  });
+
+  it("Create a new pet to the store", async () => {
+    const response = await axios.post(
+      "https://petstore.swagger.io/v2/pet",
```
Comment (Alex19777) [inline Database/test/specs/database.test.js]: Move common part of URL to some constant

Comment (Alex19777) [reply Database/test/specs/database.test.js]: And to separate class

File: Database/test/pageobjects/login.page.js (hunk 2 above)
Comment (DVA09) [reply Database/test/pageobjects/login.page.js]: Delete it

File: Database/test/specs/database.test.js (hunk 3 above)
Comment (DVA09) [reply Database/test/specs/database.test.js]: Fixed it

File: Database/test/specs/database.test.js (hunk 4 above)
Comment (DVA09) [reply Database/test/specs/database.test.js]: Fixed it

File: Database/test/specs/database.test.js (hunk 1 above)
Comment (DVA09) [reply Database/test/specs/database.test.js]: Couldn't implement it

File: Database/test/utils/url.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Alex19777) [inline Database/test/utils/url.json]: Name file like "ConfigurationData"

Comment (DVA09) [reply Database/test/utils/url.json]: Renamed file

**Review by Alex19777 — APPROVED** [review]

**Pull request merged**